lib/rubocop/cop/rails/find_each.rb

```diff
@@ -3,8 +3,12 @@
 module RuboCop
   module Cop
     module Rails
-      # Identifies usages of `all.each` and
-      # change them to use `all.find_each` instead.
+      # Identifies usages of `all.each` and change them to use `all.find_each` instead.
+      #
+      # @safety
+      #   This cop is unsafe if the receiver object is not an Active Record object.
+      #   Also, `all.each` returns an `Array` instance and `all.find_each` returns nil,
+      #   so the return values are different.
```

Comment:
@koic I noticed there's also another case, which is the presence/absence of ordering, and whether one can order by primary key. (That's only relevant when you start batching.)

Should that be added as well?

Reply:
We have ActiveRecords with nil primary keys and replacing `each` with `find_each` on these broke our queries.